packages/jest-runner/fuzz.ts

```diff
@@ -24,19 +24,25 @@ import { JazzerWorker } from "./worker";
 import { Corpus } from "./corpus";
 import * as circus from "jest-circus";
 
-// Use jests global object definition
+// Globally track when the fuzzer is started in fuzzing mode.
+let fuzzerStarted = false;
```

Comment:
Before moving forward with the review, I have a general question about this PR: Why should we only run a single fuzz test in each jest run? In the runner, we only run a single fuzz test at a time and wait for each running test to complete or throw an error. Also, when starting libFuzzer, it resets the value profile, extra counters, and 8-bit counters maps. So it starts from a clean view of the fuzzing state. The tables of recent compares do not get reset, but we can either add this patch to our fork or simply ignore this since these tables will be quickly rewritten when the new fuzz target is executed.

Reply:
I haven't looked into the libFuzzer implementation but my understanding was that "some" internal state would not be reset correctly. 
In a quick test with multiple fuzz tests the crash file was only placed in the seed directory of the last one. Doesn't libFuzzer only write crash files when the application exits? 
Furthermore, there are assertions in the libFuzzer code checking that only one instance per process is created. We could patch those out but they are probably there for a reason: https://github.com/CodeIntelligenceTesting/llvm-project-jazzer/blob/jazzer.js/compiler-rt/lib/fuzzer/FuzzerLoop.cpp#L143

Reply:
As discussed offline, we want to honor libFuzzer's assertions and only start one instance per process.